symplyphysics/laws/kinematic/vector/centrifugal_acceleration_via_centripetal_acceleration.py

```diff
@@ -0,0 +1,52 @@
+r"""
+Centrifugal acceleration via centripetal acceleration
+=====================================================
+
+Centrifugal acceleration has the same magnitude as centripetal acceleration but is directed oppositely to it.
+
+Law
+===
+    :math:`\vec a_\text{cf} = -\vec a_\text{cp}`
+"""
+
+from symplyphysics import (
+    units,
+    validate_input,
+    validate_output,
+    Vector,
+    QuantityVector,
+    scale_vector,
+)
+
+centripetal_acceleration_: Vector
+r"""
+The acceleration experienced by a rotating body in an inertial frame
+
+Symbol:
+    :math:`\vec a_\text{cp}`
+"""
+
+centrifugal_acceleration_: Vector
+r"""
+The acceleration experienced by a body in a non-inertial, rotating frame
+
+Symbol:
+    :math:`\vec a_\text{cf}`
+"""
+
+
+def centrifugal_law(centripetal_acceleration_: Vector) -> Vector:
```

Comment:
Put documentation here. Move parameter description from above here as well. Describe input parameter.

Reply:
Hm, I followed the Doxygen way of documenting functions, but not sure if I am correct about it.

Reply:
It should be docstring or restructured text format. You can generate documentation with 
```
python docs/build.py -s symplyphysics -o docs/generated -e core
```
And open html/index.html to see the result.

Reply:
Okay, I'll take a look into restructured text.